arex-storage-core/src/main/java/com/arextest/storage/core/service/AgentWorkingService.java

```diff
@@ -140,22 +145,32 @@ private <T extends MockItem> void updateMapping(@NotNull MockCategoryType catego
         if (item.getClass() == ServletMocker.class) {
             ServletMocker servlet = (ServletMocker) item;
 
-            ServiceEntity serviceEntity = serviceRepository.queryByAppId(servlet.getAppId());
-            if (serviceEntity == null) {
-                return;
+            String appServiceKey = SERVICE_MAPPINGS_PREFIX + servlet.getAppId();
```

Comment:
A variable should be declared for "CacheKeyUtils.toUtf8Bytes(appServiceKey)" in order to create fewer Byte instances for next reuse.